sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -1015,13 +1015,13 @@ export class ServiceBusMessage implements ReceivedMessage {
  * @param operation Settle operation: complete, abandon, defer or deadLetter
  * @param isRemoteSettled Boolean indicating if the message has been settled at the remote
  */
-function throwIfMessageCannotBeSettled(
+export function throwIfMessageCannotBeSettled(
   receiver: MessageReceiver | MessageSession | undefined,
   operation: DispositionType,
   isRemoteSettled: boolean
 ): void {
   let errorMessage;
-  if (!receiver) {
+  if (!receiver || !receiver.isOpen) {
```

Comment:
I believe `isOpen` is a method and we should use `!receiver.isOpen()` instead.
`!receiver.isOpen` will always return `false`
In that case, how did your test case work?

Reply:
I did the testing with `!receiver.isOpen()`. Before committing my changes  i cleaned up everything and added the change, so accidentally forgot to add the parenthesis.